azure-mgmt-resources/src/main/java/com/azure/management/resources/models/ResourceGroupsInner.java

```diff
@@ -268,7 +270,9 @@ public void delete(String resourceGroupName) {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<SimpleResponse<ResourceGroupInner>> getWithResponseAsync(String resourceGroupName) {
-        return service.get(this.client.getHost(), resourceGroupName, this.client.getSubscriptionId(), this.client.getApiVersion());
+        return FluxUtil
+                .withContext(context -> service.get(this.client.getHost(), resourceGroupName, this.client.getSubscriptionId(), this.client.getApiVersion(), context))
+                .subscriberContext(context -> context.putAll(FluxUtil.toReactorContext(this.client.getContext().addData("Sdk-Version", this.client.getApiVersion()))));

```

Comment:
In https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy, the Sdk-Version is the version of sdk (1.31.0 etc.) instead of the api -version. For Sdk-Name, I think it should use the whole package name (azure.management.resources).

Reply:
Got it. Sdk-Name seems not the whole package name (storage.blobs is apparently not a package name).

Then I will leave both to be set from the builder (similar to host/subscriptionId, manager could set this part of the Context).

Reply:
I've tested for azure-storage-blob, 12.4.0, it use this whole artifactId in user-agent. I'm not sure if all the data-plane teams use it in user-agent.
![image](https://user-images.githubusercontent.com/15213781/75750957-4be89a00-5d60-11ea-8f61-dc22a2ae15c4.png)

Reply:
is it necessary or ok to track machine id?

Reply:
I think there is no machine id? Just OS name/version, package name/version, java version.